DETAILS.md

```diff
@@ -43,7 +41,9 @@ The extension installs the following tasks:
   and user password for a [Google Compute Engine][ComputeEngine] virtual machine
   running windows.
   - **[Install Google Cloud SDK][install-cloud-sdk]**: Finds or installs the
-  [Google Cloud SDK][CloudSdk].
+  [Google Cloud SDK][CloudSdk] on windows.
+  - **[Google Cloud SDK Tool Installer][cloud-sdk-tool]**: Installs the [Google Cloud SDK][CloudSdk] as a cached
```

Comment:
Looks like the link ([Google Cloud SDK Tool Installer][cloud-sdk-tool]) doesn't work

Reply:
Fixed.